src/main/java/org/springframework/cloud/deployer/spi/kubernetes/KubernetesAppDeployer.java

```diff
@@ -156,8 +166,9 @@ public AppStatus status(String appId) {
 	private ReplicationController createReplicationController(
 			String appId, AppDeploymentRequest request,
 			Map<String, String> idMap, int externalPort) {
-		String countProperty = request.getDeploymentProperties().get(COUNT_PROPERTY_KEY);
-		int count = (countProperty != null) ? Integer.parseInt(countProperty) : 1;
+//		String countProperty = request.getDeploymentProperties().get(COUNT_PROPERTY_KEY);
+//		int count = (countProperty != null) ? Integer.parseInt(countProperty) : 1;
+		int count = 1;
```

Comment:
We can probably remove the variable `count` and use `1` as the value for the number of `replicas` directly.